pkg/reconciler/apply.go

```diff
@@ -17,6 +17,7 @@ var (
 func sFnApply(ctx context.Context, r *fsm, s *systemState) (stateFn, *ctrl.Result, error) {
 	var isError bool
 	for _, obj := range r.Objs {
+		obj.SetNamespace(s.instance.Namespace)
```

Comment:
what about resources that contain a namespace in other fields? for example RoleBindings?
example from the `keda.yaml`:
```
apiVersion: rbac.authorization.k8s.io/v1
kind: RoleBinding
metadata:
    labels:
        app.kubernetes.io/name: keda-manager-auth-reader
        helm.sh/chart: keda-2.8.1
        app.kubernetes.io/component: keda-operator
        app.kubernetes.io/managed-by: keda-operator
        app.kubernetes.io/instance: keda-manager
        app.kubernetes.io/part-of: keda-manager
        app.kubernetes.io/version: 2.8.0
    name: keda-manager-auth-reader
    namespace: kube-system
roleRef:
    apiGroup: rbac.authorization.k8s.io
    kind: Role
    name: extension-apiserver-authentication-reader
subjects:
    - kind: ServiceAccount
      name: keda-manager
      namespace: kyma-system
```

we would have RoleBinding and ServiceAccound in the same namespace (for example default) but RoleBinding would point to SA in the kyma-system namespace

Reply:
this change was commited by accident. let me roll it back